src/coordinax/_coordinax_space_frames/galactocentric.py

```diff
@@ -16,7 +16,7 @@
 from coordinax._src.vectors.d3 import CartesianVel3D, LonLatSphericalPos
 
 ScalarAngle: TypeAlias = Shaped[u.Quantity["angle"] | u.Angle, ""]
-RotationMatrix: TypeAlias = Shaped[Array, "3 3"]
+RotateMatrix: TypeAlias = Shaped[Array, "3 3"]
```

Comment:
Hmm, was this a greedy find-and-replace issue? I think it's pretty standard naming to be "rotation matrix" https://en.wikipedia.org/wiki/Rotation_matrix

Reply:
```suggestion
RotationMatrix: TypeAlias = Shaped[Array, "3 3"]
```

Reply:
Undone